timesketch/lib/utils.py

```diff
@@ -246,12 +246,46 @@ def read_and_validate_jsonl(file_handle):
     """
     # Fields that must be present in each entry of the JSONL file.
     mandatory_fields = ['message', 'datetime', 'timestamp_desc']
-    lineno = 0
+    line_number = 0
+    max_len_line_jsonl = current_app.config.get('MAX_LENGTH_LINE_JSONL')
+    tag_reducted = current_app.config.get('TAG_REDUCTED_JSONL')
     for line in file_handle:
-        lineno += 1
+        line_number += 1
         try:
             linedict = json.loads(line)
             ld_keys = linedict.keys()
+
+            current_len = len(line)
+            if current_len > max_len_line_jsonl:
+                field_len={}
+                total_len=0
+                for x in ld_keys:
+                    if isinstance(linedict[x], str):
+                        field_len[x]=len(linedict[x])
+                        total_len+=field_len[x]
+                for x in ld_keys:
```

Comment:
```suggestion
                for key in ld_keys:
```

Reply:
why loop over this twice? you can also just say 

```python
for key, key_length in field_len.items():
```

and not needing to call up `field_len[key]` all the time